test/bundlers.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const { test } = require('tap')
+const fp = require('../plugin')
+
+test('webpack removes require.main.filename', (t) => {
+  const filename = require.main.filename
+  const info = console.info
+  t.tearDown(() => {
+    require.main.filename = filename
+    console.info = info
+  })
+
+  require.main.filename = null
+
+  console.info = function (msg) {
+    t.fail('logged: ' + msg)
```

Comment:
Shouldn't `t.end()` be within this function?

Reply:
This will automatically fail the test.

Reply:
Oh. I mis-read the test.

🤔 shouldn't there at least be a `t.pass` somewhere in this test?

Reply:
Maybe, there is not really a need.